### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# units
+# cf_units
 
-A fork of iris units module
+Units of measure.
```
Comment (rhattersley) [inline README.md]: How about: `Units of measure as required by the Climate and Forecast (CF) metadata conventions.`?

**Comment by ocefpaf** [comment]
@rhattersley yep, better.

**Comment by rhattersley** [comment]
At some point you might want to make your own fork of cf_units so that the branches you're submitting as PRs are your own and can't be accidentally trampled on by others.

**Comment by ocefpaf** [comment]
Just did that.  The next PR should be OK.

**Pull request merged**